docs/install/install-broker-with-pubsub-channel.md

```diff
@@ -15,6 +15,19 @@ Channel.
 
 ## Deployment
 
+1.  Verity the configmap `config-br-default-channel` is in the `cloud-run-events` namespace.
+    ```shell
+    kubectl get configmap config-br-default-channel -n cloud-run-events
+    ```
+    By default, it is assumed to use default secret. Modify [`config-br-default-channel`](config-br-default-channel.yaml) 
+    to use different method and apply it.
+       1. If you are using workload identity, update `serviceAccount` with the Pub/Sub enabled service account you created in [Create a Pub/Sub enabled Service Account](./pubsub-service-account.md).
+        
+       1. If you are using non-default secret, update `project` and `secret`.
```

Comment:
nit: If you are using standard Kubernetes secrets, but want to use a non-default one, update `secret` with your own secret.